pkgs/servers/cloud9/default.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv ? (import <nixpkgs> {}).stdenv
+, lib ? (import <nixpkgs> {}).pkgs.lib
+, pkgs ? (import <nixpkgs> {}).pkgs
+, nodePackages ? (import <nixpkgs> {}).pkgs.nodePackages
+}:
+let
+inherit nodePackages;
+inherit lib;
+inherit pkgs;
```

Comment:
Another point is that you don't in general want to be importing any sort of angle-bracketed path inside nixpkgs, as it might be pointing to a different set of packages from the ones you're working on and lead to surprising behavior. Think of those angle brackets as a way for external (mutable) state to creep into your expressions, and we generally don't want that.